src/main/CommandHandler.cpp

```diff
@@ -568,6 +568,24 @@ CommandHandler::upgrades(std::string const& params, std::string& retStr)
             parseOptionalParam<uint32>(retMap, "protocolversion");
         p.mFlags = parseOptionalParam<uint32>(retMap, "flags");
 
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+        auto configXdrIter = retMap.find("configupgradesetkey");
+        if (configXdrIter != retMap.end())
```

Comment:
need to update docs (grep for `upgradetime`)

Reply:
Updated docs